prisma/schema.prisma

```diff
@@ -27,4 +28,10 @@ model User {
   events        Event[]
 }
 
-
+model Atendees {
+  id            Int       @id @default(autoincrement())
+  name          String?
+  event_name    Event?    @relation(fields: [eventId], references: [id])
+  eventId       Int?
+  email         String?   @unique
```

Comment:
It's not needed that this field is unique here. We can create new attendees with the same email addresses.

In the future we will have more fields on the attendee table, like for example "Accepted Event". So when there are multiple events it's possible that you've accepted one event but not the other. It's also possible that you use a different name on one event vs on the other event

Reply:
changes implemented